### DISCUSSION TITLE
cobbler/cobbler can not login

### DISCUSSION THREAD

**Discussion by michael2299** [cobbler/cobbler-web#38] [Q&A] [answered]
what is the username and password to login the website

**Comment by SchoolGuy** [comment, marked as answer]
So `cobbler` and `cobbler` should work per default. However this new webinterface is not usable yet to be honest. There is a lot missing before we can even tag the first alpha release...

**Reply by SchoolGuy** [reply]
@michael2299 were you able to login to the webinterface or do you have more problems atm with this?

**Reply by SchoolGuy** [reply]
So I tried this again today and it works. Its not pretty but it works. On the old Django interface I also have no reports of this not working, thus accepting as answer.
